### PR TITLE
feat: Disable some message menu items for encrypted messages.

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Proposed changes (including videos or screenshots)

Currently, it is not possible to properly share the content of encrypted messages outside the room.The forwarded message, message link, or quote appears in the destination room as a hash.
So, it is needed to make clear to users that it is not possible to share encrypted messages content.

image

## Further comments

[E2EE2-25](https://rocketchat.atlassian.net/browse/E2EE2-25)

[E2EE2-25]: https://rocketchat.atlassian.net/browse/E2EE2-25?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/plenty-buses-kneel.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@rocket.chat/i18n': patch
+'@rocket.chat/meteor': patch
+---
+
+Disable some message menu items for encrypted messages as they don't apply to encrypted messages and also disable apps menu items and show a warning.
```
Comment (hugocostadev) [inline .changeset/plenty-buses-kneel.md:6]: Since this is user focus, add the items that were disable, please

**Review by hugocostadev — CHANGES_REQUESTED** [review]
Missing tests 🫣

You can create just one test to check disabled items

**Review by hugocostadev — CHANGES_REQUESTED** [review]
Would be nice to have the tooltip instead of the title of the action, and a tooltip in the disabled buttons too :D 

And add tests, please 🫣

File: packages/i18n/src/locales/en.i18n.json [hunk 2]
```diff
@@ -3890,6 +3891,7 @@
   "Pexip_Enterprise_only": "Pexip (Enterprise only)",
   "Normal": "Normal",
   "Not_Available": "Not Available",
+  "Not_available_for_encrypted_messages": "Not available for encrypted messages",
```
[the remark this answers is not in the document]
Comment (yash-rajpal) [reply packages/i18n/src/locales/en.i18n.json]: Was left out by mistake, removed it.

**Review by hugocostadev — DISMISSED** [review]

**Review by hugocostadev — APPROVED** [review]

**Review by hugocostadev — APPROVED** [review]

**Pull request merged**